docs/core/build/distribution-packaging.md

```diff
@@ -107,13 +107,15 @@ The following lists the recommended packages:
   - **Version:** \<runtime version>
   - **Example:** dotnet-sdk-2.1
   - **Contains:** (3),(4)
-  - **Dependencies:** `aspnetcore-runtime-[major].[minor]`, `dotnet-targeting-pack-[major].[minor]`, `aspnetcore-targeting-pack-[major].[minor]`, `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]`, `dotnet-apphost-pack-[major].[minor]`, `dotnet-templates-[major].[minor]`
+  - **Dependencies:** `dotnet-runtime-[major].[minor]`, `aspnetcore-runtime-[major].[minor]`, `dotnet-targeting-pack-[major].[minor]`, `aspnetcore-targeting-pack-[major].[minor]`, `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]`, `dotnet-apphost-pack-[major].[minor]`, `dotnet-templates-[major].[minor]`
+  - **Provides shared directory:** `{dotnet_root}/sdk`, `{dotnet_root}/sdk/NuGetFallbackFolder`
```

Comment:
Should we have a note<sup>maybe a `sup`</sup> about what this means for anyone reading the doc for a distro that doesn't do directory ownership? (Only Fedora-like distros mentioned in the earlier PR.)

Might also be useful to call out `dotnet_root`, in the file section it looks like `.` means the same thing. Or is it a true identifier, and changing where `dotnet-host` is installed changes the rest?

Reply:
> Only Fedora-like distros mentioned in the earlier PR.

@omajid is the concept of directory ownership rpm specitic? Do you know if it applies to deb?

> Might also be useful to call out dotnet_root, in the file section it looks like . means the same thing. Or is it a true identifier, and changing where dotnet-host is installed changes the rest?

I've addressed this by consistently using `{dotnet_root}`